apps/masterbots.ai/components/layout/profile/profile-page-sidebar.tsx

```
@@ -1,153 +1,156 @@
+							<div className="rounded-lg">
+								<Link
+									href={`/u/${slug}/t`}
+									onClick={handleToggleThreads}
+									className={cn(
+										'flex w-full items-center justify-between px-4 py-3',
+										'hover:bg-gray-200 dark:hover:bg-mirage transition-colors duration-200',
+										isThreadsOpen || location.pathname?.includes('/t/')
+											? 'bg-gray-200 dark:bg-mirage'
+											: '',
+									)}
+									role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)